### PR TITLE
Club faq updates

### PR DESCRIPTION
**Pull request by Gmfholley**
This branch will create the clubs page and the faq.

### CODE REVIEW

**Comment by Gmfholley** [comment]
I'm ready for some feedback on the FAQ & Clubs page. @sandikbarr @shonnadorsey 
- I don't want to overwhelm the top nav.  I also understand that a Resources page was requested.  So I folded the About GWC, FAQ, and Resources under About as a drop down menu.  Let me know if there is a better way to do this.
- I removed the drop downs for Clubs.  If we need individual club pages, they will link from the Clubs page.
- I added a clubs page with a google map and some bare bones information that I think parents will want to determine which club to join.  I know that AIM and Do Space may not be ready, but I wanted to see what it looks like with more than one club.  Also, their information appears on the GWC code site when you apply to be a facilitator.  So they must have started the application process.

I added an 'Apply Now' button on this page, and we can use it to link to each club's WuFoo (or other) application form.  But we're always going to need to change the markup when application dates close or when clubs leave or join. Is club info worth putting in a json file or something?  But if we do that and the client doesn't have JS enabled, the whole page will fail.  If you don't like this, suggestions are welcome.
- I spent some time trying to make the FAQ readable, while keeping it navigable and in style with the rest of the site.  I also wrote them all.  Please feel free to make changes.  I'm sure there's awkward wording.  Maybe we also need to trim the number.

I'm not a great designer, so any suggestions are welcome.

I will be working on the resource page next.

**Comment by Gmfholley** [comment]
Now I've got a resources page ready for approval.  Take a look and let me know.  I still need to add links from the other pages to this new resources page.

**Comment by Gmfholley** [comment]
I added projects to the bottom of the clubs page.  Added links to the appropriate sections of the FAQ.  Updated the headers so that the nav works.  

Still needs favicon, some volunteer bios and pictures, links to apply for GWC, and anything else we need before committing to this on the site.

**Comment by sandikbarr** [comment]
Looks great @Gmfholley!

File: clubs.html [hunk 1]
```diff
@@ -164,21 +199,20 @@ <h3 class="sub-club">Completely Kids</h3>
...
               Omaha, NE 68102
             </address>
           </div>
           <div class="col-xs-12 col-sm-4">
-            <h4>Tuesdays, Wednesdays 4-6pm</h4>
-            <p>6th-8th grade</p>
-            <p><span>Instructors:</span> Not Announced</p>
+            <h4>Tuesdays, Thursdays 4:30-5:30pm</h4>
+            <p>6th grade</p>
+            <p><span>Instructors:</span> Erin Lasiter</p>
```
Comment (sandikbarr) [inline clubs.html:211]: Didn't know Erin was going to do a club too! Awesome. Let's be sure to get pics and bios for the AIM and Do Space people for the volunteers page too. I don't know if we need these by tomorrow, but we can do it soonish. We could put the current instructors in the top rows in that page as well.

**Comment by sandikbarr** [comment]
This looks great! Ready to merge if you ask me :)

**Comment by Gmfholley** [comment]
Made one last change to update the picture associated with Local Rainbows, update the link to Local Rainbows (broken now but will be fixed when I make hosting updates tomorrow), and updates the name of the project.

Ready to be merged, Sandi!

**Comment by sandikbarr** [comment]
👍 Thanks!

**Pull request merged**